django_mercadopago/migrations/0017_preference_quantity.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mp', '0016_update_model_meta'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preference',
+            name='quantity',
+            field=models.IntegerField(
```

Comment:
Probably best to restrict this to positive integers.

```suggestion
            field=models.PositiveIntegerField(
```